docs/kqueen.rst

```diff
@@ -152,26 +152,235 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
       - 127.0.0.0/8
       - Addresses allowed to access metrics endpoint without token
 
+Before you provision a Kubernetes cluster, you may need to deploy and configure the following external services:
 
-Backup
--------
+* Mail server, to enable members management. For example, to enable a user to invite other users by email. You can use the KQueen predefined mail service or run a new one.
 
-The only one statefull component of Kqueen is the etcd and users should follow https://coreos.com/etcd/docs/latest/v2/admin_guide.html#disaster-recovery. We are using `v2` etcd keys so example backup workflow can be:
+* Nginx proxy server, to configure ssl support, domain naming and define certificates.
 
-::
+* Prometheus server, to extend the monitoring in KQueen. You can either use a predefined Prometheus server, defined in the docker-compose.production.yml file or use an external one. For an external one, you must include the rules from kqueen/prod/prometheus to the existing Prometheus service.
 
-  # Backup etcd to directory /root/backup/ (etcd data stored in /var/lib/etcd/default)
-  etcdctl backup --data-dir /var/lib/etcd/default --backup-dir /root/backup/
 
-Recovery
+To set up the nginx server
+--------------------------
+
+1. Configure variables in '.env' file from Nginx section:
+
+   .. code-block:: bash
+
+      # Domain name for service. Should be equal with name in generated ssl-certificate
+      NGINX_VHOSTNAME=demo.kqueen.net
+      # Directory path for certificates in container.Finally it look like $NGINX_SSL_CERTIFICATE_DIR/$NGINX_VHOSTNAME
+      NGINX_SSL_CERTIFICATE_DIR=/mnt/letsencrypt
+
+2. Open the docker-compose.production.yml file for editing.
+3. Check proxy service configuration in docker-compose.production.yml. Pay attention on following variables:
+
+  1. ``VHOSTNAME`` - domain name for Kqueen service. Should be equal with domain name in generated certificates. Default: Using variable from .env file, named NGINX_VHOSTNAME
+  2. ``SSL_CERTIFICATE_DIR`` - Mapped directory for certificates forwarding into docker container. Default: Using variable from .env file, named NGINX_SSL_CERTIFICATE_DIR/NGINX_VHOSTNAME
+  3. ``SSL_CERTIFICATE_PATH`` - Path for cert+key certificate. Default: ``$SSL_CERTIFICATE_DIR/fullchain.cer``
+  4. ``SSL_CERTIFICATE_KEY_PATH`` -  Path for certificate key. Default: ``$SSL_CERTIFICATE_DIR/$VHOSTNAME.key``
+  5. ``SSL_TRUSTED_CERTIFICATE_PATH`` - Path for certificate only. Default: ``$SSL_CERTIFICATE_DIR/ca.cer``
+
+.. note::
+
+   Check that local certificates naming equal to defined in variables.
+
+4. Map volumes with certificates. Pay attention that destination path should be equal with SSL_CERTIFICATE_DIR. Example:
```

Comment:
Map the volumes with certificates. The destination path should be the same as SSL_CERTIFICATE_DIR.

Reply:
done